spec/requests/post_show_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'PostShow', type: :feature do
```

Comment:
Kindly use `type: request` instead of `feature` since you are making the request for the web pages and `request` tests for the entire stack, including rack, routing, and view layer.